Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,73 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
-            var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
+            var result = ForumPostEasy.GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
+        
+    }
+    class ForumPostEasy
+    {
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+
+            TimeSpan currentTime = TimeSpan.Parse("0");
+            for (int i = 0; i < exactPostTime.Length; i++)
+            {
+                for (int j = i + 1; j < exactPostTime.Length; j++)
+                {
+                    if (exactPostTime[i] == exactPostTime[j])
+                        if (showPostTime[i] != showPostTime[j])
+                            return "impossible";
+                }
+            }
+            for (int i = 0; i < exactPostTime.Length; i++)

```

Comment:
Recommending to use single for loop in case the same is being used at multiple places.